reference/7.0/Microsoft.PowerShell.Core/About/about_Windows_PowerShell_Compatibility.md

```diff
@@ -0,0 +1,94 @@
+---
+keywords: powershell,cmdlet
+locale: en-us
+ms.date: 1/10/2020
+online version: https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_windows_powershell_compatibility?view=powershell-7&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: about_Windows_PowerShell_Compatibility
+---
+# About Group Policy Settings
+
+## SHORT DESCRIPTION
+
+Describes the Windows PowerShell Compatibility funtionality for PowerShell
+
+## LONG DESCRIPTION
+
+Windows PowerShell Compatibility funtionality means that for modules in
```

Comment:
PowerShell 7 does not natively support all Windows PowerShell modules, particularly those in the the `%windir%\system32\WindowsPowerShell\v1.0\Modules` folder. Modules in that folder which are not marked as comatible with PowerShell Core are loaded in a background remoting session that makes use of Windows PowerShell 5.1.  When this feature is invoked, PowerShell creates the PS Remoting Session and uses implicit remoting to create proxy functions for the commands in the module. Invoking the proxy function causes PowerShell to run the command within the remoting session.

Reply:
@joeyaiello FYI - consider this proposal, when you'll be doing the text editing.